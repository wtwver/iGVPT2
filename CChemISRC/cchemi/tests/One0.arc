

                     SUMMARY OF  PM3 CALCULATION, Site No: 1079

                                                       MOPAC2016 (Version: 19.338L)
                                                       Wed Mar  4 14:24:09 2020

           Empirical Formula: H2 O  =     3 atoms

 PM3 1SCF GRAD CHARGE=0 Singlet

 Mopac file generated by Gabedit


     1SCF WAS SPECIFIED, SO BFGS WAS NOT USED                 
     SCF FIELD WAS ACHIEVED                                   

          HEAT OF FORMATION       =        -53.43270 KCAL/MOL =    -223.56244 KJ/MOL
          TOTAL ENERGY            =       -324.90735 EV
          ELECTRONIC ENERGY       =       -474.27017 EV
          CORE-CORE REPULSION     =        149.36282 EV
          GRADIENT NORM           =          0.65369 = 0.37741 PER ATOM
          DIPOLE                  =          1.73858 DEBYE    POINT GROUP:       C2v 
          NO. OF FILLED LEVELS    =          4
          IONIZATION POTENTIAL    =         12.316247 EV
          HOMO LUMO ENERGIES (EV) =        -12.316  4.058
          MOLECULAR WEIGHT        =         18.0152
          COSMO AREA              =         42.43 SQUARE ANGSTROMS
          COSMO VOLUME            =         25.15 CUBIC ANGSTROMS

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H     2    H     1     1.53633
            O     3    H     1     0.56094
            H     1    H     2     0.00000
          SCF CALCULATIONS        =          1
          WALL-CLOCK TIME         =          0.020 SECONDS
          COMPUTATION TIME        =          0.020 SECONDS


          FINAL GEOMETRY OBTAINED
 PM3 1SCF GRAD CHARGE=0 Singlet

 Mopac file generated by Gabedit
  H     0.00000000 +1   0.77870100 +1  -0.44376600 +1
  H     0.00000000 +1  -0.75754000 +1  -0.46039600 +1
  O     0.00000000 +1   0.00472800 +1   0.10883100 +1
 
