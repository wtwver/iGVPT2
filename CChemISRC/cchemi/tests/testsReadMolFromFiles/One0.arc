

                     SUMMARY OF PM7 CALCULATION, Site No: 1079

                                                       MOPAC2012 (Version: 15.146L)
                                                       Mon Feb 15 16:39:12 2016
                                                       No. of days left = 100

           Empirical Formula: H2 O  =     3 atoms

PM7 1SCF GRAD CHARGE=0 Singlet

 Mopac file generated by Gabedit


     1SCF WAS SPECIFIED, SO BFGS WAS NOT USED                 
     SCF FIELD WAS ACHIEVED                                   

          HEAT OF FORMATION       =        157.98325 KCAL/MOL =     661.00190 KJ/MOL
          TOTAL ENERGY            =       -313.32189 EV
          ELECTRONIC ENERGY       =       -402.93276 EV
          CORE-CORE REPULSION     =         89.61087 EV
          GRADIENT NORM           =        245.94408
          DIPOLE                  =          3.47304 DEBYE    POINT GROUP:       C2v 
          NO. OF FILLED LEVELS    =          4
          IONIZATION POTENTIAL    =          9.315872 EV
          HOMO LUMO ENERGIES (EV) =         -9.316 -0.277
          MOLECULAR WEIGHT        =         18.015
          COSMO AREA              =         56.18 SQUARE ANGSTROMS
          COSMO VOLUME            =         34.69 CUBIC ANGSTROMS

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H     3    H     2     2.96784
            H     2    O     1     1.00825
            O     1    H     2     0.00000
          SCF CALCULATIONS        =          1
          WALL-CLOCK TIME         =          0.016 SECONDS
          COMPUTATION TIME        =          0.016 SECONDS


          FINAL GEOMETRY OBTAINED
PM7 1SCF GRAD CHARGE=0 Singlet

 Mopac file generated by Gabedit
  O     0.00000000 +1   0.00000000 +1   0.20165100 +1
  H     0.00000000 +1   1.48392000 +1  -0.80660200 +1
  H     0.00000000 +1  -1.48392000 +1  -0.80660200 +1
 
