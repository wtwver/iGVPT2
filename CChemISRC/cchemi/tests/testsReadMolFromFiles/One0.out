 *******************************************************************************
 ** Site#: 1079         For non-commercial use only     Version 15.146L 64BITS**
 *******************************************************************************
 ** Cite this work as:                                                        **
 **                                                                           **
 ** (a) MOPAC2012, James J. P. Stewart, Stewart Computational Chemistry,      **
 **  Version 15.146L web: HTTP://OpenMOPAC.net         Days remaining: 100    **
 **                                                                           **
 **                                 AND                                       **
 **                                                                           **
 ** (b) Maia, J.D.C., et al., J. Chem. Theory Comput. 2012, 8, 3072-3081.     **
 **                DOI: 10.1021/ct3004645.                                    **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2012                                  **
 **                                                                           **
 *******************************************************************************

                            PM7 CALCULATION RESULTS


 *******************************************************************************
 *  CALCULATION DONE:                                Mon Feb 15 16:39:12 2016  *
 *  PM7        - The PM7 Hamiltonian to be used
 *  SINGLET    - SPIN STATE DEFINED AS A SINGLET
 *
 *
 *
 *                 CHARGE ON SYSTEM =     0
 *
 *
 *
 *  1SCF       - DO 1 SCF AND THEN STOP 
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *  GRADIENTS- ALL GRADIENTS TO BE PRINTED
 *******************************************************************************
PM7 1SCF GRAD CHARGE=0 Singlet

 Mopac file generated by Gabedit
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       O          0.00000000  *   0.00000000  *   0.20165100  *
     2       H          0.00000000  *   1.48392000  *  -0.80660200  *
     3       H          0.00000000  *  -1.48392000  *  -0.80660200  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM         X         Y         Z

     1         O        0.0000    0.0000    0.2017
     2         H        0.0000    1.4839   -0.8066
     3         H        0.0000   -1.4839   -0.8066

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1007/s00894-012-1667-x

           Empirical Formula: H2 O  =     3 atoms



      MOLECULAR POINT GROUP   :   C2v 


 SINGLET STATE CALCULATION


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS =    4


 -------------------------------------------------------------------------------
PM7 1SCF GRAD CHARGE=0 Singlet

 Mopac file generated by Gabedit


     1SCF WAS SPECIFIED, SO BFGS WAS NOT USED                 
     SCF FIELD WAS ACHIEVED                                   


                              PM7 CALCULATION
                                                       MOPAC2012 (Version: 15.146L)
                                                       Mon Feb 15 16:39:12 2016
                                                       No. of days left = 100




          FINAL HEAT OF FORMATION =        157.98325 KCAL/MOL =     661.00190 KJ/MOL


          TOTAL ENERGY            =       -313.32189 EV
          ELECTRONIC ENERGY       =       -402.93276 EV  POINT GROUP:     C2v 
          CORE-CORE REPULSION     =         89.61087 EV
          COSMO AREA              =         56.18 SQUARE ANGSTROMS
          COSMO VOLUME            =         34.69 CUBIC ANGSTROMS

          GRADIENT NORM           =        245.94408
          IONIZATION POTENTIAL    =          9.315872 EV
          HOMO LUMO ENERGIES (EV) =         -9.316 -0.277
          NO. OF FILLED LEVELS    =          4
          MOLECULAR WEIGHT        =         18.015

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H     3    H     2     2.96784
            H     2    O     1     1.00825
            O     1    H     2     0.00000


          SCF CALCULATIONS        =          1
 
          WALL-CLOCK TIME         =          0.016 SECONDS
          COMPUTATION TIME        =          0.016 SECONDS



       FINAL  POINT  AND  DERIVATIVES

   PARAMETER     ATOM    TYPE            VALUE       GRADIENT
      1          1  O    CARTESIAN X     0.000000    -0.000000  KCAL/ANGSTROM
      2          1  O    CARTESIAN Y     0.000000    -0.000000  KCAL/ANGSTROM
      3          1  O    CARTESIAN Z     0.201651   148.017224  KCAL/ANGSTROM
      4          2  H    CARTESIAN X     0.000000    -0.000000  KCAL/ANGSTROM
      5          2  H    CARTESIAN Y     1.483920   117.526259  KCAL/ANGSTROM
      6          2  H    CARTESIAN Z    -0.806602   -74.013335  KCAL/ANGSTROM
      7          3  H    CARTESIAN X     0.000000    -0.000000  KCAL/ANGSTROM
      8          3  H    CARTESIAN Y    -1.483920  -117.526260  KCAL/ANGSTROM
      9          3  H    CARTESIAN Z    -0.806602   -74.003889  KCAL/ANGSTROM




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       O          0.00000000  *   0.00000000  *   0.20165100  *
     2       H          0.00000000  *   1.48392000  *  -0.80660200  *
     3       H          0.00000000  *  -1.48392000  *  -0.80660200  *

                             CARTESIAN COORDINATES

   1    O        0.000000000     0.000000000     0.201651000
   2    H        0.000000000     1.483920000    -0.806602000
   3    H        0.000000000    -1.483920000    -0.806602000


           Empirical Formula: H2 O  =     3 atoms



      MOLECULAR POINT GROUP   :   C2v 


                  EIGENVALUES  
 -22.36746 -10.66642 -10.04148  -9.31587  -0.27670   0.51498


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          O          -0.702416        6.7024     1.99250     4.70992
    2          H           0.351208        0.6488     0.64879
    3          H           0.351208        0.6488     0.64879
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.000     0.000    -3.402     3.402
 HYBRID        -0.000     0.000    -0.071     0.071
 SUM           -0.000     0.000    -3.473     3.473


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz   
    1  O   1.99250   2.00000   1.33396   1.37596
    2  H   0.64879
    3  H   0.64879

          JOB ENDED NORMALLY



 TOTAL JOB TIME:             0.02 SECONDS

 == MOPAC DONE ==
